src/KeyedFrames.jl

```diff
@@ -161,6 +161,24 @@ function delete!(kf::KeyedFrame, ind::Vector{<:Symbol})
     return kf
 end
 
+##### RENAME #####
+
+function rename!(kf::KeyedFrame, nms)
+    rename!(kf.frame, nms)
+
+    for (from, to) in nms
+        kf.key[kf.key .== from] = to
```

Comment:
Probably more efficient to do:
```
i = findfirst(kf.key, from)
kf.key[i] = to
```
so that you aren't allocating a bit array on each iteration and you have an early exit condition. Probably isn't a big deal given that you probably won't have hundreds of columns.